src/sentry/features/__init__.py

```diff
@@ -67,7 +67,6 @@
 default_manager.add("organizations:dashboards-manage", OrganizationFeature, True)
 default_manager.add("organizations:data-forwarding", OrganizationFeature)
 default_manager.add("organizations:discover", OrganizationFeature)
-default_manager.add("organizations:discover-arithmetic", OrganizationFeature, True)
```

Comment:
Do the flags need to be removed in a follow-up PR to make sure everything goes well on the frontend as well?